cmd/api/src/test/integration/harnesses.go

```diff
@@ -1082,6 +1082,204 @@ func (s *SearchHarness) Setup(graphTestContext *GraphTestContext) {
 	graphTestContext.UpdateNode(s.GroupLocalGroup)
 }
 
+type ADCSESC1Harness struct {
```

Comment:
Please include the harness svg and json representation(s) under cmd/api/src/test/integration/harnesses for reference